examples/experimental/dagster-blueprints/dagster_blueprints/cli.py

```diff
@@ -55,6 +70,49 @@ def generate_schemas(loader_module: Optional[str] = None):
         click.echo(f"Wrote schema for {loader.per_file_blueprint_type} to {schema_path}")
 
 
+@click.command(
+    help="Generates and installs a VS Code extension which provides JSON schemas for Blueprint types specified by YamlBlueprintsLoader objects."
+)
+@click.option(
+    "--loader-module",
+    type=click.STRING,
+    help="Path of Python module that contains YamlBlueprintsLoader objects. Defaults to Dagster project module, if `pyproject.toml` exists.",
+)
+@click.option(
+    "--vscode-folder-path",
+    type=click.STRING,
+    help="Path to the VS Code workspace folder. If not provided, attempts to find the .vscode directory in the current directory or any of its parents.",
```

Comment:
Is "workspace folder" accurate VS Code terminology? I find the terminology a bit mind boggling: https://stackoverflow.com/questions/44629890/what-is-a-workspace-in-visual-studio-code.

Reply:
I think the right term is "workspace root folder" - there are multi-root workspaces, but a `.vscode` directory lives in that case in each root: 

https://code.visualstudio.com/docs/editor/multi-root-workspaces

> With multiple root folders in one workspace, it is possible to have a .vscode folder in each root folder defining the settings that should apply for that folder.